teuthology/task/install.py

```diff
@@ -577,6 +529,23 @@ def remove_sources(ctx, config):
             p.spawn(remove_fn, remote, project)
 
 
+def get_package_list(config):
+    debug = config.get('debuginfo', False)
+    project = config.get('project', 'ceph')
+    yaml_path = os.path.join(os.path.dirname(__file__), 'packages.yaml')
```

Comment:
The mere presence of this file, which avoids teuthology having a hard dependency on ceph-qa-suite, is not problematic. This feature allows an alternate package list to be provided; one way to do that would be shipping one in whichever ceph-qa-suite branches need it. Currently, as I understand it, that is just jewel and master right now.

What you're proposing, if I understand it correctly, is to remove the default entirely. So for that reason alone, teuthology would become dependent on ceph-qa-suite. And further, we would have to add a package list to every ceph-qa-suite branch that currently exists.

I honestly don't see how that would be a good thing.